src/Core/WebServer.cs

```diff
@@ -386,6 +387,54 @@ public async Task ViewExtensionScript(HttpContext context)
         await context.Response.CompleteAsync();
     }
 
+    /// <summary>Web route for audio files.</summary>
+    public async Task ViewAudio(HttpContext context)
+    {
+        string path = context.Request.Path.ToString();
+        if (path.StartsWith("/Audio/"))
```

Comment:
redundant check - you copied this from view output, which only has it because there are two different routes pathed to it, this one doesn't have that